test cases/common/220 fs module/meson.build

```diff
@@ -15,7 +15,7 @@ if not is_windows and build_machine.system() != 'cygwin'
   # * Windows user permissions to create symlinks, and/or Windows in Developer mode
   # so at this time the symlink test is skipped for Windows.
   symlink = meson.current_build_dir() / 'a_symlink'
-  run_command('ln', '-s', meson.current_source_dir() / 'meson.build', symlink)
+  run_command('ln', '-s', meson.current_source_dir() / 'meson.build', symlink, check: true)
```

Comment:
This is a great error right here!

The test runner configures this test case correctly, and it succeeds in generating, running, and in theory passing. But the test runner ***also*** checks that the backend will correctly force regeneration for build/test, by updating the utime of meson.build.

On this second run, the `ln` program fails because
```
--- stderr ---
/usr/bin/ln: failed to create symbolic link '/tmp/b d19af95f3b/a_symlink': File exists
```

Reply:
(after your latest push)

Note that the additional `-f` flag to ln would make it overwrite the existing symlink rather than erroring.

Reply:
Done